hadoop-ozone/dev-support/intellij/runConfigurations/OzoneFsShell.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="OzoneFsShell" type="Application" factoryName="Application">
+    <envs>
+      <env name="OZONE_CONF_DIR" value="$PROJECT_DIR$/hadoop-ozone/dev-support/intellij" />
+    </envs>
+    <option name="MAIN_CLASS_NAME" value="org.apache.hadoop.fs.ozone.OzoneFsShell" />
+    <module name="hadoop-ozone-filesystem" />
+    <option name="PROGRAM_PARAMETERS" value="-conf=hadoop-ozone/dev-support/intellij/ozone-site.xml -ls -R o3fs://bucket1.vol1/" />
+    <option name="VM_PARAMETERS" value="-Dlog4j.configuration=file:hadoop-ozone/dev-support/intellij/log4j.properties -Xbootclasspath/a:$PROJECT_DIR$/hadoop-ozone/dev-support/intellij -Xbootclasspath/a:$MAVEN_REPOSITORY$/commons-cli/commons-cli/1.2/commons-cli-1.2.jar" />
```

Comment:
```suggestion
    <option name="VM_PARAMETERS" value="-Dlog4j.configuration=file:hadoop-ozone/dev-support/intellij/log4j.properties -Xbootclasspath/a:$PROJECT_DIR$/hadoop-ozone/dev-support/intellij" />
```

Reply:
Committing suggestion to remove explicit inclusion of jar files that are now automatically included in classpath with change of module from '_ozone-filesystem_' to '_ozone-tools_'.